src/DynamoRevit/RevitPathResolver.cs

```diff
@@ -62,7 +62,9 @@ internal RevitPathResolver(string userDataFolder, string commonDataFolder)
                 if (File.Exists(assemblyPath))
                 {
                     preloadLibraryPaths.Add(assemblyPath);
-                    additionalNodeDirectories.Add(Path.GetDirectoryName(assemblyPath));
+                    string assemblyDir = Path.GetDirectoryName(assemblyPath);
+                    additionalNodeDirectories.Add(assemblyDir);
+                    additionalResolutionPaths.Add(assemblyDir);
```

Comment:
i added this line because Dynamo need to know from where to load the xml files with the documentation for the nodes